backend/courses/filters.py

```diff
@@ -201,7 +219,8 @@ def filter_choices(queryset, choices):
 class CourseSearchFilterBackend(filters.BaseFilterBackend):
     def filter_queryset(self, request, queryset, view):
         filters = {
-            "requirements": requirement_filter,
+            "attributes": attribute_filter,
+            "pre_ngss_requirements": pre_ngss_requirement_filter,
```

Comment:
We actually don't need to remove this (though the requirements filter wouldn't behave as expected). I can just remove this line if that's a good solution.

Reply:
Ig since the requirements filter is disabled on the frontend currently it's fine to keep this change. We just have to remember to change the request schema when we re-enable the filter.